src/interfaces/api/Title.ts

```diff
@@ -0,0 +1,161 @@
+// Generated by https://quicktype.io
+
+export interface TitleData {
```

Comment:
It's generated, but we don't need all of the properties, stick to the ones that we need